examples/app.py

```diff
@@ -123,4 +124,5 @@ def current_user():
 (Also includes directive to leverage pythons threading capacity.)
 '''
 if __name__ == '__main__':
-	app.run(threaded=True, port=int(os.environ.get("PORT", 8080)))
+	    app.run(threaded=True, port=int(os.environ.get("PORT", (re.findall("^\d+$|(?<=:)\d+$", os.environ.get("SPOTIPY_REDIRECT_URI", 8080)))[0])))
```

Comment:
Not a fan of the difficult regex here. To get what follows the last colon, could we not do:

    redirect_uri.split(":")[-1]

Reply:
also indentation needs fixing

Reply:
probably, but the regex not only matches the port in SPOTIPY_REDIRECT_URI but also the default 8080
I don't know of the top of my head how we could do this as a oneliner.

Reply:
If we do it with more than one line, the best readable version probably would be 
`    port = 8080`
`    port = int(os.environ.get("SPOTIPY_REDIRECT_URI").split(":")[-1]) if os.environ.get("PORT").split(":")[-1] else port`
`    port = int(os.environ.get("PORT")) if os.environ.get("PORT") else port`
`    app.run(threaded=True, port=port)`

Reply:
never mind, I figured out a way. 
But still, it does not look nice, but at least is more readable then the ont-the-fly fix I previously stated

Reply:
There is no need to specifically catch numbers, since we expect something that looks like DOMAIN:PORT

```python
app.run(threaded=True, port=int(os.environ.get("PORT",
                                os.environ.get("SPOTIPY_REDIRECT_URI", 8080).split(":")[-1])))
```

This defaults to 8080, is still a one-liner, is shorter and simpler to understand.

Reply:
That looks very nice, thanks!